x-pack/plugins/cloud_security_posture/public/components/page_template.tsx

```diff
@@ -45,28 +51,46 @@ const defaultProps: KibanaPageTemplateProps = {
   template: 'default',
 };
 
-interface CspPageTemplateProps extends KibanaPageTemplateProps {
-  isLoading?: boolean;
-  loadingText?: string;
-}
+const NO_DATA_CONFIG: KibanaPageTemplateProps['noDataConfig'] = {
+  pageTitle: NO_DATA_CONFIG_TITLE,
+  solution: NO_DATA_CONFIG_SOLUTION_NAME,
+  // TODO: Add real docs link once we have it
+  docsLink: 'https://www.elastic.co/guide/index.html',
+  logo: 'logoSecurity',
+  actions: {
+    elasticAgent: {
+      // TODO: Use `href` prop to link to our own integration once we have it
+      title: NO_DATA_CONFIG_BUTTON,
+      description: NO_DATA_CONFIG_DESCRIPTION,
+    },
+  },
+};
+
+export const CspPageTemplate: React.FC<KibanaPageTemplateProps> = ({ children, ...props }) => {
```

Comment:
change file name to match this exported component

Reply:
We'll change all `Csp` variables soon enough, waiting until then